dp/cloud/python/magma/radio_controller/tests/unit/test_dp_logs.py

```diff
@@ -26,8 +27,10 @@
 SOME_SERIAL_NUMBER = 'some_serial_number'
 SOME_NETWORK_ID = 'some_network_id'
 CBSD_REGISTER = 'CBSDRegister'
+SOME_DATE = '2022-05-20T00:00:00+00:00'
 
 
+@freeze_time('2022-05-20')
```

Comment:
Should this be a reference to `SOME_DATE`? Also, given that this date is today, this might accidentally work only today, maybe choose a date in the past instead.